app/controllers/katello/api/v2/package_groups_controller.rb

```diff
@@ -18,6 +58,12 @@ def default_sort
       %w(name asc)
     end
 
+    protected
+
+    def find_repository
+      @repository = Repository.find(params[:repository_id])
+    end
```

Comment:
Although that base module uses @repo, so you'd need to change your references to taht.